### PR TITLE
#86 bytes serializer

### PR DESCRIPTION
**Pull request by denmusic1992**
add bytes encoder, also add tests, they have been passed successfully.

### CODE REVIEW

**Review by y9san9 — APPROVED** [review]

**Review by y9san9 — APPROVED** [review]

File: web3/src/commonMain/kotlin/dev.icerock.moko.web3/contract/BytesParam.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package dev.icerock.moko.web3.contract
```
Comment (anton6tak) [inline web3/src/commonMain/kotlin/dev.icerock.moko.web3/contract/BytesParam.kt:1]: Copyright please

File: web3/src/commonMain/kotlin/dev.icerock.moko.web3/contract/StringParam.kt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package dev.icerock.moko.web3.contract
```
Comment (anton6tak) [inline web3/src/commonMain/kotlin/dev.icerock.moko.web3/contract/StringParam.kt:1]: Copyright please

File: web3/src/commonTest/kotlin/dev.icerock.moko.web3/TestEncoder.kt [hunk 3]
```diff
@@ -0,0 +1,38 @@
+package dev.icerock.moko.web3
```
Comment (anton6tak) [inline web3/src/commonTest/kotlin/dev.icerock.moko.web3/TestEncoder.kt:1]: Copyright please

**Comment by anton6tak** [comment]
Update moko-web3 version in Readme and libs.versions.toml please

File: web3/src/commonMain/kotlin/dev.icerock.moko.web3/contract/BytesParam.kt (hunk 1 above)
Comment (denmusic1992) [reply web3/src/commonMain/kotlin/dev.icerock.moko.web3/contract/BytesParam.kt:5]: done

File: web3/src/commonMain/kotlin/dev.icerock.moko.web3/contract/StringParam.kt (hunk 2 above)
Comment (denmusic1992) [reply web3/src/commonMain/kotlin/dev.icerock.moko.web3/contract/StringParam.kt:5]: done

File: web3/src/commonTest/kotlin/dev.icerock.moko.web3/TestEncoder.kt (hunk 3 above)
Comment (denmusic1992) [reply web3/src/commonTest/kotlin/dev.icerock.moko.web3/TestEncoder.kt:5]: done

**Pull request merged**